src/os_win/os_fs.c

```diff
@@ -527,12 +507,6 @@ __win_open_file(WT_FILE_SYSTEM *file_system, WT_SESSION *wt_session, const char
         if (win_fh->filehandle == INVALID_HANDLE_VALUE) {
             windows_error = __wt_getlasterror();
             ret = __wt_map_windows_error(windows_error);
-            __wt_err(session, ret,
```

Comment:
Put back but without the win_fh->direct_io ?, i.e. keep only the : part.

Reply:
Hi Tod, addressed here [a98162b](https://github.com/wiredtiger/wiredtiger/pull/10430/commits/a98162bc3cb1aa0cffb47e3175091a48c7551a8b)